config.go

```diff
@@ -269,7 +269,6 @@ type config struct {
 
 	Profile string `long:"profile" description:"Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65535"`
 
-	DebugHTLC          bool   `long:"debughtlc" description:"Activate the debug htlc mode. With the debug HTLC mode, all payments sent use a pre-determined R-Hash. Additionally, all HTLCs sent to a node with the debug HTLC R-Hash are immediately settled in the next available state transition."`
```

Comment:
Just something to note -- I've seen `lnd.conf`s with the option in it, so removing it will prevent some people from starting, in which case they'll see an error along the lines of `unknown option: debughtlc`.

Reply:
I didn't think about that. But we have removed or renamed flags before. I don't know what the procedure was there, but it seems to me that a note in the release notes should be enough. Especially for this flag that has little use for real users.

Reply:
We've renamed config options in the past (`noencryptwallet` -> `noseedbackup`) without much issue. I also would wager that there aren't manny nodes in production that're running with this flag in the wild, and if they are, then this update will be a wake up call for t hem.

Reply:
we disabled the use of `debughtlc` on mainnet [over a year ago](https://github.com/lightningnetwork/lnd/pull/1475). if people still have this enabled, they already can't start

Reply:
They could've just flipped it from true to false. Nothing major though, just wanted to point it out.

Reply:
good point, yes that's certainly possible too :)